docs/osindex.html

```diff
@@ -138,163 +196,169 @@
+      var coords = circle.geometry.coordinates[0].join(' ');
+
+      // Create an OGC XML filter parameter value which will select the localBuildings
+      // features intersecting the circle polygon coordinates.
+      var xml = '<ogc:Filter>';
+      xml += '<ogc:Intersects>';
+      xml += '<ogc:PropertyName>SHAPE</ogc:PropertyName>';
+      xml += '<gml:Polygon srsName="urn:ogc:def:crs:EPSG::4326">';
+      xml += '<gml:outerBoundaryIs>';
+      xml += '<gml:LinearRing>';
+      xml += '<gml:coordinates>' + coords + '</gml:coordinates>';
+      xml += '</gml:LinearRing>';
+      xml += '</gml:outerBoundaryIs>';
+      xml += '</gml:Polygon>';
+      xml += '</ogc:Intersects>';
+      xml += '</ogc:Filter>';
+
+      // Define parameters object.
+      var wfsParams = {
+        key: apiKey,
+        service: 'WFS',
+        request: 'GetFeature',
+        version: '2.0.0',
+        typeNames: 'Zoomstack_LocalBuildings',
+        outputFormat: 'GEOJSON',
+        srsName: 'urn:ogc:def:crs:EPSG::4326',
+        filter: xml,
+        count: 100,
+        startIndex: 0
+      };
+
+      var resultsRemain = true;
+
+      geojson.features.length = 0;
+
+      map.getSource('localBuildings').setData(geojson);
+      document.getElementById('feature-count').innerHTML = '';
+      document.getElementById('area').innerHTML = '';
+
+      // Use fetch() method to request GeoJSON data from the OS Features API.
+      //
+      // If successful - set the GeoJSON data for the 'localBuildings' layer and re-render
+      // the map.
+      //
+      // Calls will be made until the number of features returned is less than the
+      // requested count, at which point it can be assumed that all features for
+      // the query have been returned, and there is no need to request further pages.
+      function fetchWhile(resultsRemain) {
+        if (resultsRemain) {
+          fetch(getUrl(wfsParams))
+            .then(response => response.json())
+            .then((data) => {
+              wfsParams.startIndex += wfsParams.count;
+
+              geojson.features.push.apply(geojson.features, data.features);
+
+              resultsRemain = data.features.length < wfsParams.count ? false : true;
+
+              fetchWhile(resultsRemain);
+            });
+        } else {
+          map.getSource('localBuildings').setData(geojson);
+          document.getElementById('feature-count').innerHTML = geojson.features.length;
+
+          // To calculate the area, rounded to 3 decimal places
+          let area = 0;
+          let ratio = 0;
+          let circle_area = Math.PI * 0.5 * 0.5;
+          let feature = geojson.features;
+          for (let i = 0; i < feature.length; i++) {
+            area += geojson.features[i].properties.SHAPE_Area
+            ratio = area / circle_area
+
+          }
+          document.getElementById('area').innerHTML = (area / 1000000).toFixed(2);
```

Comment:
instead of changing at the presentation layer, express your variable in the right unit.